internal/core/analysis_api/handlers/list.go

```diff
@@ -120,6 +122,21 @@ func pythonListFilters(
 		filters = append(filters, tagFilter)
 	}
 
+	if initialSet != nil {
+		if *initialSet {
+			initialSetFilter := expression.Equal(expression.Name("createdBy"),
+				expression.Value("00000000-0000-4000-8000-000000000000"))
+
+			filters = append(filters, initialSetFilter)
+		}
+		if !*initialSet {
```

Comment:
This should be an `else` clause, you don't need to explicitly check if its value is false because you already know it isn't true.

```suggestion
		} else {
```